### PR TITLE
feat: Menu improvements

### LINKED ISSUE #1169: Menu space is limited by its opener
https://docs.cozy.io/cozy-ui/react/#!/Menu

image

I guess portaling the Menu would solve the problem.

### LINKED ISSUE #1170: Adapt black colors for Menu
Icon color and text should have slighty different shades of grey.

https://mattermost.cozycloud.cc/cozycloud/pl/93stn1zcu38wucd4z58zz6ub9r

Icon:  slate-grey
Text: charcoal-grey

### PR DESCRIPTION
**Pull request by ptbrowne**
- Fix the colors (fix #1170), charcoalGrey for text, slateGrey for icon
- Menu used in examples was the one from material-ui, styleguidist got confused when
  auto-providing the `Menu` variable inside the examples. Requiring explicitly fixes
  it (fix #1169).
- Have the menu automatically opened for Argos (provided the isArgos() util that can
  be used in examples for this kind of behavior. Based on user-agent)

Preview here : https://ptbrowne.github.io/cozy-ui/react/#/Menu

### CODE REVIEW

File: react/Menu/index.jsx [hunk 1]
```diff
@@ -124,10 +131,10 @@ class Menu extends Component {
         {...restProps}
       >
         {!component ? (
-          <MenuButton
+          <Button
             disabled={disabled}
             onClick={this.toggle}
-            label={text}
+            label={text || label}
             buttonClassName={buttonClassName}
```
Comment (Crash--) [inline react/Menu/index.jsx]: are we sure that we have in this `buttonClassName` the equal of : `className={cx('c-btn', styles['c-menu__btn'], buttonClassName)}` ? 🤔 

you say that there is no style but what about : 
```
.c-menu__btn
    font-size rem(16)
    border 0
    cursor pointer

.c-menu__btn:not([disabled]):not([aria-disabled=true]):hover
.c-menu__btn:not([disabled]):not([aria-disabled=true]):active
.c-menu__btn:not([disabled]):not([aria-disabled=true]):focus
    background-color transparent
```

File: react/Menu/index.jsx [hunk 2]
```diff
@@ -175,8 +181,10 @@ Menu.propTypes = {
   onSelectDisabled: PropTypes.func,
   /** Global Styles for MenuItems */
   itemsStyle: PropTypes.object,
-  /** if you need fixed menu */
-  popover: PropTypes.bool
+  /** If you need fixed menu */
+  popover: PropTypes.bool,
+  /** Whether the menu should be initially opened */
+  initialOpen: PropTypes.bool
```
Comment (Crash--) [inline react/Menu/index.jsx]: SelectBox has a `menuIsOpen` prop. Should we keep the same name?

File: docs/utils.js [hunk 3]
```diff
@@ -40,3 +40,7 @@ export const fakeIntentCreate = (action, doctype, options) => {
 export const t = (path) => {
   return get(translations, path)
 }
+
+export const isArgos = () => {
+  return navigator.userAgent.includes('Argos')
+}
```
Comment (Crash--) [inline docs/utils.js]: It seems that we have a `isTesting` variable to take "open" snapshot. Should we set `isTesting` to true when we are in a jest context but also when the userAgent is Argos? Like that no need to do : 

`menuIsOpen={isArgos() || isTesting}`

File: react/Menu/index.jsx (hunk 2 above)
Comment (ptbrowne) [reply react/Menu/index.jsx]: I don't think so, menuIsOpen controls the component but here we do not control it. It is just the initial value.

File: docs/utils.js (hunk 3 above)
Comment (ptbrowne) [reply docs/utils.js]: Correct, I made the change, thanks

File: react/Menu/index.jsx (hunk 1 above)
Comment (ptbrowne) [reply react/Menu/index.jsx]: I removed the commit removing the MenuButton from this PR.

**Pull request merged**